### ISSUE TITLE
Breaking change after 2024.6.4

### ISSUE THREAD

**Issue by ADeadPixel** [EverythingSmartHome/everything-presence-one#200]
Whole device became unavailable after updating to latest ESPHome.
Current info from EP1:
ESPHome Firmware: 2024.7.0
Hardware Rev: 1.4

I get the following logs when attempting to connect via wifi
```txt
INFO ESPHome 2024.7.0
INFO Reading configuration /config/esphome/ep1-0.yaml...
WARNING GPIO15 is a strapping PIN and should only be used for I/O with care.
Attaching external pullup/down resistors to strapping pins can cause unexpected failures.
See https://esphome.io/guides/faq.html#why-am-i-getting-a-warning-about-strapping-pins
INFO Starting log output from 192.168.20.213 using esphome API
WARNING Can't connect to ESPHome API for ep1-0 @ 192.168.20.213: Error connecting to [AddrInfo(family=<AddressFamily.AF_INET: 2>, type=<SocketKind.SOCK_STREAM: 1>, proto=6, sockaddr=IPv4Sockaddr(address='192.168.20.213', port=6053))]: [Errno 111] Connect call failed ('192.168.20.213', 6053) (SocketAPIError)
INFO Trying to connect to ep1-0 @ 192.168.20.213 in the background
INFO Successfully connected to ep1-0 @ 192.168.20.213 in 0.121s
WARNING ep1-0 @ 192.168.20.213: Connection error occurred: ep1-0 @ 192.168.20.213: Connection lost
INFO Successfully connected to ep1-0 @ 192.168.20.213 in 0.063s
WARNING ep1-0 @ 192.168.20.213: Connection error occurred: ep1-0 @ 192.168.20.213: Connection lost
INFO Successfully connected to ep1-0 @ 192.168.20.213 in 0.194s
WARNING ep1-0 @ 192.168.20.213: Connection error occurred: ep1-0 @ 192.168.20.213: Connection lost
INFO Successfully connected to ep1-0 @ 192.168.20.213 in 0.153s
WARNING ep1-0 @ 192.168.20.213: Connection error occurred: ep1-0 @ 192.168.20.213: Connection lost
```

and plugging in directly I get the following

```txt
[14:48:57]h�����F�HF±�[E][esp-idf:000][wifi]: I (7745) wifi:
[E][esp-idf:000][wifi]: <ba-add>idx:1 (ifx:0, da:21:f9:83:3c:cf), tid:0, ssn:0, winSize:64
[E][esp-idf:000][wifi]: 

E (11600) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
[14:48:57]E (11600) task_wdt:  - loopTask (CPU 1)
[14:48:57]E (11600) task_wdt: Tasks currently running:
[14:48:57]E (11600) task_wdt: CPU 0: IDLE0
[14:48:57]E (11600) task_wdt: CPU 1: IDLE1
[14:48:57]E (11600) task_wdt: Aborting.
[14:48:57]
[14:48:57]abort() was called at PC 0x400f99c8 on core 0
[14:48:57]
[14:48:57]
[14:48:57]Backtrace: 0x40082966:0x3ffb0860 0x400896e5:0x3ffb0880 0x4008f89a:0x3ffb08a0 0x400f99c8:0x3ffb0910 0x400832e9:0x3ffb0930 0x4016ce6b:0x3ffbad90 0x400fa119:0x3ffbadb0 0x4008aabc:0x3ffbadd0
[14:48:57]
[14:48:57]
[14:48:57]
[14:48:57]
[14:48:57]ELF file SHA256: f98735db8beff4fd
[14:48:57]
[14:48:57]Rebooting...
[14:48:57]ets Jul 29 2019 12:21:46
[14:48:57]
[14:48:57]rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
[14:48:57]configsip: 0, SPIWP:0xee
[14:48:57]clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
[14:48:57]mode:DIO, clock div:2
[14:48:57]load:0x3fff0030,len:6652
[14:48:57]ho 0 tail 12 room 4
[14:48:57]load:0x40078000,len:15052
[14:48:57]load:0x40080400,len:3836
[14:48:57]entry 0x4008069c
[14:48:57]I (29) boot: ESP-IDF 4.4.7 2nd stage bootloader
[14:48:57]I (29) boot: compile time 13:03:59
[14:48:57]I (29) boot: Multicore bootloader
[14:48:57]I (32) boot: chip revision: v3.1
[14:48:57]I (36) boot.esp32: SPI Speed      : 40MHz
[14:48:57]I (41) boot.esp32: SPI Mode       : DIO
[14:48:57]I (45) boot.esp32: SPI Flash Size : 4MB
[14:48:57]I (50) boot: Enabling RNG early entropy source...
[14:48:57]I (55) boot: Partition Table:
[14:48:57]I (59) boot: ## Label            Usage          Type ST Offset   Length
[14:48:57]I (66) boot:  0 otadata          OTA data         01 00 00009000 00002000
[14:48:57]I (73) boot:  1 phy_init         RF data          01 01 0000b000 00001000
[14:48:57]I (81) boot:  2 app0             OTA app          00 10 00010000 001c0000
[14:48:57]I (88) boot:  3 app1             OTA app          00 11 001d0000 001c0000
[14:48:57]I (96) boot:  4 nvs              WiFi data        01 02 00390000 0006d000
[14:48:57]I (103) boot: End of partition table
[14:48:57]I (108) esp_image: segment 0: paddr=00010020 vaddr=3f400020 size=300d8h (196824) map
[14:48:58]I (187) esp_image: segment 1: paddr=00040100 vaddr=3ffb0000 size=03350h ( 13136) load
[14:48:58]I (193) esp_image: segment 2: paddr=00043458 vaddr=40080000 size=0cbc0h ( 52160) load
[14:48:58]I (215) esp_image: segment 3: paddr=00050020 vaddr=400d0020 size=a03a8h (656296) map
[14:48:58]I (452) esp_image: segment 4: paddr=000f03d0 vaddr=4008cbc0 size=08370h ( 33648) load
[14:48:58]I (477) boot: Loaded app from partition at offset 0x10000
[14:48:58]I (477) boot: Disabling RNG early entropy source...
[14:48:58]I (488) cpu_start: Multicore app
[14:48:58]I (488) cpu_start: Pro cpu up.
[14:48:58]I (489) cpu_start: Starting app cpu, entry point is 0x40081f94
[14:48:58]I (477) cpu_start: App cpu up.
[14:48:58]I (507) cpu_start: Pro cpu start user code
[14:48:58]I (507) cpu_start: cpu freq: 160000000
[14:48:58]I (507) cpu_start: Application information:
[14:48:58]I (511) cpu_start: Project name:     ep1-0
[14:48:58]I (516) cpu_start: App version:      2024.7.0
[14:48:58]I (521) cpu_start: Compile time:     Jul 17 2024 13:03:25
[14:48:58]I (527) cpu_start: ELF file SHA256:  f98735db8beff4fd...
[14:48:58]I (533) cpu_start: ESP-IDF:          4.4.7
[14:48:58]I (538) cpu_start: Min chip rev:     v0.0
[14:48:58]I (543) cpu_start: Max chip rev:     v3.99 
[14:48:58]I (547) cpu_start: Chip rev:         v3.1
[14:48:58]I (552) heap_init: Initializing. RAM available for dynamic allocation:
[14:48:58]I (559) heap_init: At 3FFAE6E0 len 00001920 (6 KiB): DRAM
[14:48:58]I (565) heap_init: At 3FFB7C90 len 00028370 (160 KiB): DRAM
[14:48:58]I (572) heap_init: At 3FFE0440 len 00003AE0 (14 KiB): D/IRAM
[14:48:58]I (578) heap_init: At 3FFE4350 len 0001BCB0 (111 KiB): D/IRAM
[14:48:58]I (584) heap_init: At 40094F30 len 0000B0D0 (44 KiB): IRAM
[14:48:58]I (592) spi_flash: detected chip: generic
[14:48:58]I (595) spi_flash: flash io: dio
[14:48:58]I (602) cpu_start: Starting scheduler on PRO CPU.
[14:48:58]I (0) cpu_start: Starting scheduler on APP CPU.
[14:49:09][E][esp-idf:000]: I (238) gpio: GPIO[25]| InputEn: 1| OutputEn: 1| OpenDrain: 1| Pullup: 1| Pulldown: 0| Intr:0 

[E][esp-idf:000]: I (239) gpio: GPIO[26]| InputEn: 1| OutputEn: 1| OpenDrain: 1| Pullup: 1| Pulldown: 0| Intr:0 

[E][esp-idf:000]: I (273) uart: queue free spaces: 20

[E][esp-idf:000]: I (274) gpio: GPIO[32]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 

[E][esp-idf:000]: I (275) gpio: GPIO[15]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 0| Pulldown: 1| Intr:0 

[E][shtcx:080]: sensor polling failed
[E][esp-idf:000]: I (293) gpio: GPIO[33]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 0| Pulldown: 1| Intr:0 

[E][esp-idf:000]: I (310) wifi:
[E][esp-idf:000]: wifi driver task: 3ffc80e0, prio:23, stack:6656, core=0
[E][esp-idf:000]: 

[E][esp-idf:000][wifi]: I (1227) system_api: Base MAC address is not set

[E][esp-idf:000][wifi]: I (1235) system_api: read default base MAC address from EFUSE

[E][esp-idf:000][wifi]: I (1249) wifi:
[E][esp-idf:000][wifi]: wifi firmware version: 1fd20f4
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1266) wifi:
[E][esp-idf:000][wifi]: wifi certification version: v7.0
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1276) wifi:
[E][esp-idf:000][wifi]: config NVS flash: enabled
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1297) wifi:
[E][esp-idf:000][wifi]: config nano formating: disabled
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1307) wifi:
[E][esp-idf:000][wifi]: Init data frame dynamic rx buffer num: 32
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1327) wifi:
[E][esp-idf:000][wifi]: Init static rx mgmt buffer num: 5
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1348) wifi:
[E][esp-idf:000][wifi]: Init management short buffer num: 32
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1358) wifi:
[E][esp-idf:000][wifi]: Init dynamic tx buffer num: 32
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1379) wifi:
[E][esp-idf:000][wifi]: Init static rx buffer size: 1600
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1399) wifi:
[E][esp-idf:000][wifi]: Init static rx buffer num: 10
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1409) wifi:
[E][esp-idf:000][wifi]: Init dynamic rx buffer num: 32
[E][esp-idf:000][wifi]: 

[E][esp-idf:000]: I (1431) wifi_init: rx ba win: 6

[E][esp-idf:000]: I (1440) wifi_init: tcpip mbox: 32

[E][esp-idf:000]: I (1450) wifi_init: udp mbox: 6

[E][esp-idf:000]: I (1450) wifi_init: tcp mbox: 6

[E][esp-idf:000]: I (1460) wifi_init: tcp tx win: 5760

[E][esp-idf:000]: I (1461) wifi_init: tcp rx win: 5760

[E][esp-idf:000]: I (1471) wifi_init: tcp mss: 1440

[E][esp-idf:000]: I (1481) wifi_init: WiFi IRAM OP enabled

[E][esp-idf:000]: I (1481) wifi_init: WiFi RX IRAM OP enabled

[E][esp-idf:000][wifi]: I (1492) phy_init: phy_version 4791,2c4672b,Dec 20 2023,16:06:06

[E][esp-idf:000][wifi]: I (1576) wifi:
[E][esp-idf:000][wifi]: mode : sta (d4:d4:da:28:d1:90)
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1577) wifi:
[E][esp-idf:000][wifi]: enable tsf
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1582) wifi:
[E][esp-idf:000][wifi]: Set ps type: 1

[E][esp-idf:000][wifi]: 

[E][shtcx:094]: sensor read failed
[E][esp-idf:000][wifi]: I (2611) wifi:
[E][esp-idf:000][wifi]: Set ps type: 1

[E][esp-idf:000][wifi]: 

[E][shtcx:094]: sensor read failed
[E][esp-idf:000][wifi]: I (5209) wifi:
[E][esp-idf:000][wifi]: new:<6,2>, old:<1,0>, ap:<255,255>, sta:<6,2>, prof:1
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (5423) wifi:
[E][esp-idf:000][wifi]: state: init -> auth (b0)
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (5438) wifi:
[E][esp-idf:000][wifi]: state: auth -> assoc (0)
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (5454) wifi:
[E][esp-idf:000][wifi]: state: assoc -> run (10)
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (5500) wifi:
[E][esp-idf:000][wifi]: connected with RPFP - IoT, aid = 1, channel 6, 40D, bssid = da:21:f9:83:3c:cf
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (5502) wifi:
[E][esp-idf:000][wifi]: security: WPA2-PSK, phy: bgn, rssi: -41
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (5517) wifi:
[E][esp-idf:000][wifi]: pm start, type: 1

[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (5563) wifi:
[E][esp-idf:000][wifi]: <ba-add>idx:0 (ifx:0, da:21:f9:83:3c:cf), tid:6, ssn:2, winSize:64
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (5575) wifi:
[E][esp-idf:000][wifi]: AP's beacon interval = 102400 us, DTIM period = 1
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][sys_evt]: I (6532) esp_netif_handlers: sta ip: 192.168.20.213, mask: 255.255.255.0, gw: 192.168.20.1
```

My issues seemed related to https://github.com/EverythingSmartHome/everything-presence-one/issues/198 but I have never had any issues with the connection and my IoT VLAN can talk to the outside world to receive updates. 

I have already wiped my install and tried a fresh install which resulted in this 

```txt
[15:23:31]E (11189) task_wdt: Task watchdog got triggered. The following tasks did not reset the watchdog in time:
[15:23:31]E (11189) task_wdt:  - loopTask (CPU 1)
[15:23:31]E (11189) task_wdt: Tasks currently running:
[15:23:31]E (11189) task_wdt: CPU 0: IDLE0
[15:23:31]E (11189) task_wdt: CPU 1: IDLE1
[15:23:31]E (11189) task_wdt: Aborting.
[15:23:31]
[15:23:31]abort() was called at PC 0x40101d34 on core 0
[15:23:31]
[15:23:31]
[15:23:31]Backtrace: 0x40082966:0x3ffb0890 0x400896e5:0x3ffb08b0 0x4008f89a:0x3ffb08d0 0x40101d34:0x3ffb0940 0x400832e9:0x3ffb0960 0x40175d6f:0x3ffbadc0 0x40102485:0x3ffbade0 0x4008aabc:0x3ffbae00
[15:23:31]
[15:23:31]
[15:23:31]
[15:23:31]
[15:23:31]ELF file SHA256: 9ce2528cd65b87c8
[15:23:31]
[15:23:31]Rebooting...
[15:23:31]ets Jul 29 2019 12:21:46
[15:23:31]
[15:23:31]rst:0xc (SW_CPU_RESET),boot:0x13 (SPI_FAST_FLASH_BOOT)
[15:23:31]configsip: 0, SPIWP:0xee
[15:23:31]clk_drv:0x00,q_drv:0x00,d_drv:0x00,cs0_drv:0x00,hd_drv:0x00,wp_drv:0x00
[15:23:31]mode:DIO, clock div:2
[15:23:31]load:0x3fff0030,len:6652
[15:23:31]ho 0 tail 12 room 4
[15:23:31]load:0x40078000,len:15052
[15:23:31]load:0x40080400,len:3836
[15:23:31]entry 0x4008069c
[15:23:31]I (29) boot: ESP-IDF 4.4.7 2nd stage bootloader
[15:23:31]I (29) boot: compile time 15:06:52
[15:23:31]I (29) boot: Multicore bootloader
[15:23:31]I (32) boot: chip revision: v3.1
[15:23:31]I (36) boot.esp32: SPI Speed      : 40MHz
[15:23:31]I (41) boot.esp32: SPI Mode       : DIO
[15:23:31]I (45) boot.esp32: SPI Flash Size : 4MB
[15:23:31]I (50) boot: Enabling RNG early entropy source...
[15:23:31]I (55) boot: Partition Table:
[15:23:31]I (59) boot: ## Label            Usage          Type ST Offset   Length
[15:23:31]I (66) boot:  0 otadata          OTA data         01 00 00009000 00002000
[15:23:31]I (73) boot:  1 phy_init         RF data          01 01 0000b000 00001000
[15:23:31]I (81) boot:  2 app0             OTA app          00 10 00010000 001c0000
[15:23:31]I (88) boot:  3 app1             OTA app          00 11 001d0000 001c0000
[15:23:31]I (96) boot:  4 nvs              WiFi data        01 02 00390000 0006d000
[15:23:31]I (103) boot: End of partition table
[15:23:31]I (108) esp_image: segment 0: paddr=00010020 vaddr=3f400020 size=37f08h (229128) map
[15:23:31]I (199) esp_image: segment 1: paddr=00047f30 vaddr=3ffb0000 size=03380h ( 13184) load
[15:23:31]I (205) esp_image: segment 2: paddr=0004b2b8 vaddr=40080000 size=04d60h ( 19808) load
[15:23:31]I (213) esp_image: segment 3: paddr=00050020 vaddr=400d0020 size=a92ach (692908) map
[15:23:31]I (464) esp_image: segment 4: paddr=000f92d4 vaddr=40084d60 size=101d0h ( 66000) load
[15:23:31]I (502) boot: Loaded app from partition at offset 0x10000
[15:23:31]I (502) boot: Disabling RNG early entropy source...
[15:23:31]I (513) cpu_start: Multicore app
[15:23:31]I (513) cpu_start: Pro cpu up.
[15:23:31]I (514) cpu_start: Starting app cpu, entry point is 0x40081f94
[15:23:31]I (502) cpu_start: App cpu up.
[15:23:31]I (532) cpu_start: Pro cpu start user code
[15:23:31]I (532) cpu_start: cpu freq: 160000000
[15:23:31]I (532) cpu_start: Application information:
[15:23:31]I (536) cpu_start: Project name:     ep1-0
[15:23:31]I (541) cpu_start: App version:      2024.7.0
[15:23:31]I (546) cpu_start: Compile time:     Jul 17 2024 15:06:17
[15:23:31]I (552) cpu_start: ELF file SHA256:  9ce2528cd65b87c8...
[15:23:31]I (558) cpu_start: ESP-IDF:          4.4.7
[15:23:31]I (563) cpu_start: Min chip rev:     v0.0
[15:23:31]I (567) cpu_start: Max chip rev:     v3.99 
[15:23:31]I (572) cpu_start: Chip rev:         v3.1
[15:23:31]I (577) heap_init: Initializing. RAM available for dynamic allocation:
[15:23:31]I (584) heap_init: At 3FFAE6E0 len 00001920 (6 KiB): DRAM
[15:23:31]I (590) heap_init: At 3FFB7CC0 len 00028340 (160 KiB): DRAM
[15:23:31]I (597) heap_init: At 3FFE0440 len 00003AE0 (14 KiB): D/IRAM
[15:23:31]I (603) heap_init: At 3FFE4350 len 0001BCB0 (111 KiB): D/IRAM
[15:23:31]I (609) heap_init: At 40094F30 len 0000B0D0 (44 KiB): IRAM
[15:23:31]I (617) spi_flash: detected chip: generic
[15:23:31]I (620) spi_flash: flash io: dio
[15:23:31]I (626) cpu_start: Starting scheduler on PRO CPU.
[15:23:31]I (0) cpu_start: Starting scheduler on APP CPU.
[15:23:41][E][esp-idf:000]: I (241) gpio: GPIO[25]| InputEn: 1| OutputEn: 1| OpenDrain: 1| Pullup: 1| Pulldown: 0| Intr:0 

[E][esp-idf:000]: I (242) gpio: GPIO[26]| InputEn: 1| OutputEn: 1| OpenDrain: 1| Pullup: 1| Pulldown: 0| Intr:0 

[E][esp-idf:000]: I (276) uart: queue free spaces: 20

[E][esp-idf:000]: I (278) gpio: GPIO[32]| InputEn: 0| OutputEn: 1| OpenDrain: 0| Pullup: 0| Pulldown: 0| Intr:0 

[E][esp-idf:000]: I (278) gpio: GPIO[15]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 0| Pulldown: 1| Intr:0 

[E][shtcx:080]: sensor polling failed
[E][esp-idf:000]: I (297) gpio: GPIO[33]| InputEn: 1| OutputEn: 0| OpenDrain: 0| Pullup: 0| Pulldown: 1| Intr:0 

[E][esp-idf:000]: I (313) wifi:
[E][esp-idf:000]: wifi driver task: 3ffc814c, prio:23, stack:6656, core=0
[E][esp-idf:000]: 

[E][esp-idf:000][wifi]: I (1258) system_api: Base MAC address is not set

[E][esp-idf:000][wifi]: I (1259) system_api: read default base MAC address from EFUSE

[E][esp-idf:000][wifi]: I (1271) wifi:
[E][esp-idf:000][wifi]: wifi firmware version: 1fd20f4
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1298) wifi:
[E][esp-idf:000][wifi]: wifi certification version: v7.0
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1309) wifi:
[E][esp-idf:000][wifi]: config NVS flash: enabled
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1329) wifi:
[E][esp-idf:000][wifi]: config nano formating: disabled
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1340) wifi:
[E][esp-idf:000][wifi]: Init data frame dynamic rx buffer num: 32
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1360) wifi:
[E][esp-idf:000][wifi]: Init static rx mgmt buffer num: 5
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1380) wifi:
[E][esp-idf:000][wifi]: Init management short buffer num: 32
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1391) wifi:
[E][esp-idf:000][wifi]: Init dynamic tx buffer num: 32
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1411) wifi:
[E][esp-idf:000][wifi]: Init static rx buffer size: 1600
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1432) wifi:
[E][esp-idf:000][wifi]: Init static rx buffer num: 10
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1442) wifi:
[E][esp-idf:000][wifi]: Init dynamic rx buffer num: 32
[E][esp-idf:000][wifi]: 

[E][esp-idf:000]: I (1463) wifi_init: rx ba win: 6

[E][esp-idf:000]: I (1463) wifi_init: tcpip mbox: 32

[E][esp-idf:000]: I (1483) wifi_init: udp mbox: 6

[E][esp-idf:000]: I (1483) wifi_init: tcp mbox: 6

[E][esp-idf:000]: I (1493) wifi_init: tcp tx win: 5760

[E][esp-idf:000]: I (1493) wifi_init: tcp rx win: 5760

[E][esp-idf:000]: I (1503) wifi_init: tcp mss: 1440

[E][esp-idf:000]: I (1513) wifi_init: WiFi IRAM OP enabled

[E][esp-idf:000]: I (1514) wifi_init: WiFi RX IRAM OP enabled

[E][esp-idf:000][wifi]: I (1525) phy_init: phy_version 4791,2c4672b,Dec 20 2023,16:06:06

[E][esp-idf:000][wifi]: I (1613) wifi:
[E][esp-idf:000][wifi]: mode : sta (d4:d4:da:28:d1:90)
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1615) wifi:
[E][esp-idf:000][wifi]: enable tsf
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (1619) wifi:
[E][esp-idf:000][wifi]: Set ps type: 1

[E][esp-idf:000][wifi]: 

[E][shtcx:094]: sensor read failed
[E][esp-idf:000][wifi]: I (2649) wifi:
[E][esp-idf:000][wifi]: Set ps type: 1

[E][esp-idf:000][wifi]: 

[E][shtcx:094]: sensor read failed
[E][esp-idf:000][wifi]: I (4653) wifi:
[E][esp-idf:000][wifi]: new:<6,2>, old:<1,0>, ap:<255,255>, sta:<6,2>, prof:1
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (4867) wifi:
[E][esp-idf:000][wifi]: state: init -> auth (b0)
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (4917) wifi:
[E][esp-idf:000][wifi]: state: auth -> assoc (0)
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (4933) wifi:
[E][esp-idf:000][wifi]: state: assoc -> run (10)
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (4982) wifi:
[E][esp-idf:000][wifi]: connected with RPFP - IoT, aid = 1, channel 6, 40D, bssid = da:21:f9:83:3c:cf
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (4983) wifi:
[E][esp-idf:000][wifi]: security: WPA2-PSK, phy: bgn, rssi: -34
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (4998) wifi:
[E][esp-idf:000][wifi]: pm start, type: 1

[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (5010) wifi:
[E][esp-idf:000][wifi]: AP's beacon interval = 102400 us, DTIM period = 1
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][wifi]: I (5055) wifi:
[E][esp-idf:000][wifi]: <ba-add>idx:0 (ifx:0, da:21:f9:83:3c:cf), tid:6, ssn:2, winSize:64
[E][esp-idf:000][wifi]: 

[E][esp-idf:000][sys_evt]: I (6013) esp_netif_handlers: sta ip: 192.168.20.213, mask: 255.255.255.0, gw: 192.168.20.1
```

**Comment by SasStu** [comment]
Same here

**Comment by thetonto** [comment]
Confirmed.  Device appears back as default and also back to older version once you adopt it.

**Comment by mannerisms** [comment]
Also same here, tried re-flashing over usb but nothing seems to help, all my ESP one devices have gone dead and become unavailable to homeassistant

**Comment by EverythingSmartHome** [comment]
Can you confirm if it works flashing from the user guide: https://everythingsmarthome.github.io/everything-presence-one/updating.html

This version is on 2024.6.6

**Comment by ADeadPixel** [comment]
> Can you confirm if it works flashing from the user guide: https://everythingsmarthome.github.io/everything-presence-one/updating.html
> 
> This version is on 2024.6.6

That is where I attempted to re-flash from.

**Comment by EverythingSmartHome** [comment]
Just so we can rule it out, is it possible for you to connect the WiFi to your regular SSID with internet access?

**Comment by ADeadPixel** [comment]
Sure thing, I swapped it over to the primary network but still nothing in HA. For context the EP1 can still communicate via WiFi (I can still upload wirelessly) but nothing is in the log and nothing is in HA.
```txt
INFO ESPHome 2024.7.0
INFO Reading configuration /config/esphome/ep1-0.yaml...
WARNING GPIO15 is a strapping PIN and should only be used for I/O with care.
Attaching external pullup/down resistors to strapping pins can cause unexpected failures.
See https://esphome.io/guides/faq.html#why-am-i-getting-a-warning-about-strapping-pins
INFO Starting log output from 192.168.1.166 using esphome API
INFO Successfully connected to ep1-0 @ 192.168.1.166 in 0.040s
INFO Successful handshake with ep1-0 @ 192.168.1.166 in 0.087s
```

Here is how it looks in HA. Notice how it shows old information even though the logs show it was ESPHome version 2024.7.0 above.
![image](https://github.com/user-attachments/assets/630701ec-0364-4937-ad36-f4da4fbb3fc9)

**Comment by TheHackmeister** [comment]
I've been having the same issue end of this week. Ended up rolling back the VM running ESPHome to try and flash with  2024.6.4 and 2024.6.6 but encountered the same. Before that I had a backup of 2024.6.2 but that version of ESPHome was incompatible with the EP package. Figured fine, I'll pin the package to an earlier version. Turns out, the issues is likely in package version v1.2.4 as  v1.2.3 works fine on ESPHome 2024.7.0. So this sort of thing at least brought my sensors back into a usable state: 
```
packages:
  athom.epo-ble: github://everythingsmarthome/presence-one/everything-presence-one-ble.yaml@v1.2.3
```

Hope it helps!

**Comment by ADeadPixel** [comment]
I can confirm updating to the package listed did address the problem. So something is wrong with the v1.2.4 release and >2024.6.3 ESPHome.

<img width="1055" alt="image" src="https://github.com/user-attachments/assets/812584ee-853a-46f8-9cc1-ad4b6ddff2c8">

Here is my config for anyone who needs a reference. If you do revert to 1.2.3 make sure to add in the OTA information.

```yaml
substitutions:
  name: ep1-0
  friendly_name: Robert's Office EP1

packages:
  # Everything_Smart_Technology.Everything_Presence_One: github://everythingsmarthome/everything-presence-one/everything-presence-one.yaml@1.2.3
  testing: github://everythingsmarthome/presence-one/everything-presence-one-ble.yaml@v1.2.3

esphome:
  name: ${name}
  name_add_mac_suffix: false
  friendly_name: ${friendly_name}

ota:
  platform: esphome

api:
  encryption:
    key: !secret encryption_key
    
wifi:
  ssid: !secret wifi_ssid
  password: !secret wifi_password
```

**Comment by EverythingSmartHome** [comment]
Thanks for the details. The only real difference in those versions is the new update mechanism which does make me wonder if it can't access the update file and reboots constantly trying to keep checking for updates.

**Comment by SasStu** [comment]
Hi I can confirm this. Just installed the EP One in my IoT network and its working, back to my NoT network and we are back to red leds...

**Comment by mannerisms** [comment]
Same here, after following the discussion here, the NoT network was the issue.

**Comment by SasStu** [comment]
OK I just added

update: !remove

as a workaround to my configuration and now the device is working again in my NoT network.

**Comment by EverythingSmartHome** [comment]
This should be resolved in https://github.com/EverythingSmartHome/everything-presence-one/pull/206, please update to version 1.0.7. A fix is coming in ESPHome 2024.7.2 but this workaround does the same thing for now.

**Comment by taintedkernel** [comment]
@EverythingSmartHome Thank you, that update solved it! I spent a few days trying to troubleshoot what was wrong with my new device on my NoT HASS network.
